Bots/DotNet/Consumers/Composer/ComposerHostBotDotNet/README.md

```diff
@@ -0,0 +1,27 @@
+# Welcome to your new bot
+
```

Comment:
The readme was removed for the rest of the bots. We should remove this one as well.